From 268cd22fc42fd333a42b2524bf52c75ebc91b901 Mon Sep 17 00:00:00 2001
From: Jerry Hung <jerry.hung@gumstix>
Date: Mon, 12 Dec 2016 10:57:28 -0800
Subject: [PATCH] Removing IRQF_DISALBE

Kernel has stopped support the function of IRQD_DISABLE due to many significant reasons.

Remove IRQD_DISABLE from the pwm.c

Signed-off-by: Jerry Hung <jerry@gumsix.com>
---
 pwm.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/pwm.c b/pwm.c
index f2be64f..3f10e4f 100644
--- a/pwm.c
+++ b/pwm.c
@@ -332,7 +332,7 @@ static int pwm_timer_init(void)
			pwm_dev[i].irq = omap_dm_timer_get_irq(pwm_dev[i].timer);

			if (request_irq(pwm_dev[i].irq, match_handler,
-					IRQF_DISABLED | IRQF_SHARED,
+					IRQF_SHARED,
					"pwm-match",
					&pwm_dev[i])) {
				printk(KERN_ERR
--
1.9.1

